### PR TITLE
k8s/mesos datawatch support netservice ip pool resource.

### PR DESCRIPTION
**Pull request by mrmgxxxx**
k8s/mesos datawatch support netservice ip pool resource

### CODE REVIEW

File: bcs-mesos/bcs-mesos-watch/main.go [hunk 1]
```diff
@@ -57,6 +57,11 @@ func setCfg(op *MesosWatchOptions) {
 
 	cfg.KubeConfig = op.Kubeconfig
 	cfg.StoreDriver = op.StoreDriver
+
+	cfg.NetServiceZK = op.NetServiceZK
+	if len(cfg.NetServiceZK) == 0 {
+		cfg.NetServiceZK = cfg.RegDiscvSvr
```
Comment (DeveloperJim) [inline bcs-mesos/bcs-mesos-watch/main.go]: Good

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-bcs?pullRequest=306) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-bcs?pullRequest=306) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: mrmgxxxx<br/>:white_check_mark: DeveloperJim<br/>:x: meloji<br/><hr/>**meloji** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-bcs?pullRequest=306) it.</sub>

File: bcs-k8s/bcs-k8s-watch/app/app.go [hunk 2]
```diff
@@ -114,6 +113,10 @@ func Run(configFilePath string) error {
 	glog.Info("Init config DONE!")
 
 	zkHosts := strings.Join(watchConfig.BCS.ZkHosts, ",")
+	if watchConfig.BCS.NetServiceZKHosts == nil || len(watchConfig.BCS.NetServiceZKHosts) == 0 {
```
Comment (DeveloperJim) [inline bcs-k8s/bcs-k8s-watch/app/app.go]: nil就意味着len为0，可以兼容

Comment (mrmgxxxx) [reply bcs-k8s/bcs-k8s-watch/app/app.go]: 1

**Pull request merged**